backend/src/test/kotlin/cz/loono/backend/api/service/ExaminationRecordServiceTest.kt

```diff
@@ -150,6 +150,65 @@ class ExaminationRecordServiceTest(
         assertThat(actual?.badges?.first()?.type).isEqualTo("COAT")
     }
 
+    @Test
+    fun `Create correct future exam`() {
+        val uid = "101"
+        accountRepository.save(
+            createAccount(
+                uid = uid,
+                sex = SexDto.MALE.value,
+                birthday = LocalDate.of(1990, 9, 9)
+            )
+        )
+        val examinationRecordService =
+            ExaminationRecordService(
+                accountRepository,
+                examinationRecordRepository,
+                selfExaminationRecordRepository,
+                preventionService,
+                clock
+            )
+        val exam = ExaminationRecordDto(
+            type = ExaminationTypeDto.GENERAL_PRACTITIONER,
+            date = LocalDateTime.now().plusYears(2)
+        )
+
+        examinationRecordService.createOrUpdateExam(exam, uid)
+        val actual = accountRepository.findByUid(uid)
+
+        // FIXME IS points and badge calculation in this test correct? Creation of a new in future shouldn't add points.
```

Comment:
This one is fixed here [79dbab8](https://github.com/cesko-digital/loono-be/pull/61/commits/79dbab84656d6cb2e3fd636b0d3f531144c3b37a)